api/rest/v2/resolve.md

```diff
@@ -361,7 +381,7 @@ Request (GET):
 
 ```bash
```

Comment:
update the header for this example box to "GET Request" to be consistent with the rest of the page?

Also update "Request (POST)" to "POST Request"

Reply:
Another great catch, thanks!